lms/djangoapps/verify_student/views.py

```diff
@@ -594,6 +613,10 @@ def _requirements(self, display_steps, is_active):
             self.WEBCAM_REQ: False,
         }
 
+        # Remove the account activation requirement if disabled via waffle
+        if waffle.switch_is_active(DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH):
+            all_requirements.pop(self.ACCOUNT_ACTIVATION_REQ)
```

Comment:
I didn't think I needed it, but after I added the new test I observed the requirement was still being returned in some cases, so I wanted to make sure I covered all the bases.  Thinking about the overall case now, perhaps this is ALL we need to have, versus overriding the user 's active status.  Going to do some local testing to see.